app/src/main/java/net/programmierecke/radiodroid2/players/exoplayer/ExoPlayerWrapper.java

```diff
@@ -100,6 +100,9 @@ public void onReceive(Context context, Intent intent) {
         final int MIN_RETRY_DELAY_MS = 10;
         final SharedPreferences sharedPrefs = PreferenceManager.getDefaultSharedPreferences(context);
 
+        // We read need to read the retry delay here on each error again because the user might
```

Comment:
I think there is unnecessary `read` here in the comment.

Reply:
Oops, thanks.